docs/changelog.md

```diff
@@ -20,6 +20,7 @@
 
 Added:
 
+- A helper function named [fit_gaussian()][extra.utils.fit_gaussian] (!131).
```

Comment:
Can you move this up into the 'unreleased' section?

(It was probably in the right place when you made the PR, but I inserted a subheading to do a release, so now it's no longer correct.

Reply:
Oops, yes, fixed in 2af0c8d.